cluster/windows/cni/kubeproxystart.ps1

```diff
@@ -1,3 +1,18 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
```

Comment:
Does this really require a separate commit to only add the licenses?

Reply:
Squeezed this commit. Was just following the guidance to not to squeeze until the final merge, to ease the review process.

Reply:
Taking care of the other comments - removing the ps1 files & updating the commit message.